packages/p2p/src/components/my-profile/my-profile.jsx

```diff
@@ -180,10 +213,16 @@ const MyProfile = observer(() => {
                                                         my_profile_store.default_advert_description.length
                                                     }
                                                     max_characters={300}
+                                                    onFocus={e => setCurrentFocus(e.currentTarget.name)}
+                                                    onBlur={() => setCurrentFocus(null)}
                                                 />
                                             )}
                                         </Field>
-                                        <div className='my-profile__footer'>
+                                        <div
+                                            className={classNames('my-profile__footer', {
+                                                'my-profile__footer--inherit': has_on_screen_keyboard,
```

Comment:
Can we make this className similar to the on line 165, since we're checking for `has_on_screen_keyboard` in both. Hence:
```suggestion
                                                'my-profile__footer--active-keyboard': has_on_screen_keyboard,
```